tests/benchmarks/utils.py

```diff
@@ -22,3 +27,49 @@ def clean_simulated_ddp_distributed_env():
     os.environ.pop('LOCAL_RANK')
     os.environ.pop('MASTER_ADDR')
     os.environ.pop('MASTER_PORT')
+
+
+def get_free_port():
+    """Get free port."""
+    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+        s.bind(('', 0))
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        return s.getsockname()[1]
+
+
+def setup_simulated_ddp_distributed_env_custom(world_size, local_rank, port):
```

Comment:
Please modify my tests to merge setup_simulated_ddp_distributed_env function into one.

Reply:
I will modify it in another pr for revise test util